pkg/node/getBalanceOf.go

```diff
@@ -0,0 +1,31 @@
+package node
+
+import (
+	"fmt"
+	"strconv"
+)
+
+type GetBalanceOfResponse struct {
+	CandidateBalance float64
+	FinalBalance     float64
+}
+
+// GetBalanceOf returns as a float64 the candidate balance and final balance of an address
+func GetBalanceOf(client *Client, walletAddress string) (*GetBalanceOfResponse, error) {
```

Comment:
Names to improve:
- The balance is retreived remotely. You should clarify that by using fetch instead of Get
- The wallet part doesn't bring additional info.

My suggestion:
```suggestion
func FetchBalance(client *Client, address string) (*Balance, error) {
```